### ISSUE TITLE
Book 05: Sentence 073

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#33]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/e7e94c53-4cf1-4bb3-897c-53809c416bf4)

```
# sent_id = Book_05_-_The_Omens.txt:73
# text = Ic matlactli capitulo, vncan mjtoa: in tetzaujtl injc motetzaujaia, in jntechpa azcame, anoço cuecueia: ioan in qujqujmjchtin.
# text[norm] = Ic matlactli capítulo, oncan mihtoa: in tetzahuitl inic motetzahuiaya, in intechpa azcame, anozo cuecueya: ihuan in quiquimichtin.
# text[orig] = Ic·matlactli·capitulo·,·vncan·mj¶toa·:·in·tetzaujtl·injc·motetzaujaia·,·injntechpa·azcame·,·anoço·cuecueia·:·ioan·inqujqujmjchtin·.
# text[osp] = Capitulo diez, en que se tracta del aguero: que tomauan de las hormigas, y ranas, y ratones, en cierto caso.
# references = A&D/175:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 56.52%
1	Ic	ic	ADV	_	_	3	amod	_	Folio=10r|Paragraph=2|Line=1|Norm=Ic
2	matlactli	matlactli	NUM	_		1	nummod	_	Folio=10r|Paragraph=2|Line=1|Norm=matlactli|Gloss=diez|Analysed=Yes
3	capitulo	capítulo	NOUN	_	State=Abs	6	obl	_	Folio=10r|Paragraph=2|Line=1|Norm=capítulo|Analysed=Yes
4	,	,	PUNCT	_		6	punct	_	Folio=10r|Paragraph=2|Line=1|Norm=,|Analysed=Yes
5	vncan	oncan	ADV	_		6	advmod	_	Folio=10r|Paragraph=2|Line=1|Norm=oncan|Gloss=aquí|Analysed=Yes
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	Orig=mj¶·toa|Folio=10r,10r|Paragraph=2,2|Line=1,2|Norm=mihtoa|Analysed=Yes
7	:	:	PUNCT	_		9	punct	_	Folio=10r|Paragraph=2|Line=2|Norm=:|Analysed=Yes
8	in	in	DET|SCONJ	_	_	9	_	_	Folio=10r|Paragraph=2|Line=2|Norm=in
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	_	_	Folio=10r|Paragraph=2|Line=2|Norm=tetzahuitl|Analysed=Yes
10	injc	inic	SCONJ	_	_	11	_	_	Folio=10r|Paragraph=2|Line=2|Norm=inic
11	motetzaujaia	tetzahuia	VERB	_	_	9	_	_	Folio=10r|Paragraph=2|Line=2|Norm=motetzahuiaya|Ambiguous=Yes|Analysed=Yes|Guessed=Yes
12	,	,	PUNCT	_		14	punct	_	Folio=10r|Paragraph=2|Line=2|Norm=,|Analysed=Yes
13	in	in	DET|SCONJ	_	_	14	_	_	Orig=injntechpa|Folio=10r|Paragraph=2|Line=3|Norm=in
14	jntechpa	techpa	NOUN	_	NounType=Relat|Number[psor]=Plur|Person[psor]=3|State=Pos	11	_	_	Orig=injntechpa|Folio=10r|Paragraph=2|Line=3|Norm=intechpa|Analysed=Yes
15	azcame	azcatl	NOUN	_	Number=Plur	14	_	_	Folio=10r|Paragraph=2|Line=3|Norm=azcame|Gloss=hormigas|Analysed=Yes
16	,	,	PUNCT	_		18	punct	_	Folio=10r|Paragraph=2|Line=3|Norm=,|Analysed=Yes
17	anoço	anozo	ADV	_		18	_	_	Folio=10r|Paragraph=2|Line=3|Norm=anozo|Analysed=Yes
18	cuecueia	cuecueya	NOUN	_	State=Abs	15	_	_	Folio=10r|Paragraph=2|Line=3|Norm=cuecueya|Gloss=faldas|Analysed=Yes
19	:	:	PUNCT	_		20	punct	_	Folio=10r|Paragraph=2|Line=3|Norm=:|Analysed=Yes
20	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	15	_	_	Folio=10r|Paragraph=2|Line=4|Norm=ihuan|Gloss=con|Analysed=Yes
21	in	in	DET	_	_	22	det	_	Orig=inqujqujmjchtin|Folio=10r|Paragraph=2|Line=4|Norm=in
22	qujqujmjchtin	quimichin	NOUN	_	Number=Plur	20	nmod	_	Orig=inqujqujmjchtin|Folio=10r|Paragraph=2|Line=4|Norm=quiquimichtin|Gloss=ratones|Analysed=Yes
23	.	.	PUNCT	_		6	punct	_	Folio=10r|Paragraph=2|Line=4|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Questions:
- An ugly thing about treating relational nouns as the head and not having a `CCONJ` reading for `ihuan` is that *in intechpa azcame, anozo cuecueya: ihuan in quiquimichtin.* we can't just extract *azcameh, cuecueyah, quiquimichtin* as a series of coordinated NPs.
- This is a bit like #14, but does not have the *catca*... 
- *intechpa* is like in #11.

**Comment by ftyers** [comment]
López Austin:
> Décimo capítulo, donde se habla del augurio, de lo que se tenía por agüero con relación a las hormigas, o a las ranas y a los ratones.

**Comment by ftyers** [comment]
Updated:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/1dafb2ea-84af-4c69-9da6-69a371d398db)

This sentence is very similar in structure to #39, and maybe in principle to those with *in ihcuac...* too.

**Comment by Lguyogiro** [comment]
re _tetzahuitl_ and _motetzahuiaya_, this is being discussed in #39

**Comment by Lguyogiro** [comment]
> An ugly thing about treating relational nouns as the head and not having a CCONJ reading for ihuan is that in intechpa azcame, anozo cuecueya: ihuan in quiquimichtin. we can't just extract azcameh, cuecueyah, quiquimichtin as a series of coordinated NPs.

I think this is primarily an issue with this specific reln _-huan_, the meaning of which, when it has a third-person singular possessor, seems to fall variably between "with" and "and" ("along with" in Lockhart's words). 

_ihuan_ in this respect presents other issues, e.g. (4.B.3 in Lockhart)

"nictlazohtla in nonamictzin ihuan niquihtoa ca nictlazohtla in nopiltzin"

...where ihuan seems to have a verbal complement (it is worth pointing out that, afaik, _-pampa_ also can take verbal complement, which presumably we would annotate as `acl(-pampa, V)`, so we could also account for that this way.

So might we want to distinguish between these two cases, (1) typical reln behavior or (2) conj behavior, even though the boundary between them is fuzzy?

**Comment by Lguyogiro** [comment]
> This is a bit like https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/issues/14, but does not have the catca...

Yeah I think that even without _catca_ this analysis makes sense since, as I believe we discussed elsewhere, I think the clausal subject makes more sense for _mijtoa_ than to essentially say that "the portent was said".

**Comment by Lguyogiro** [comment]
regarding _in intechpa_ , since we are treating this as a clause, we probably want to select `SCONJ` and `mark`

**Comment by ftyers** [comment]
So like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/7b2a2773-5ebe-4ede-a711-6e4f538c27c0)

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Merged in [968051e](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/commit/968051e8eb2640999f2c21533a00c4c272447c94)
